sites/includes/metadata.html

```diff
@@ -1 +1,9 @@
-<meta name="description" content="{% if page.description[page.lang] %}{{ page.description[page.lang] }}{% else %}{{ site.site-description[page.lang] }} {% endif %}" />
\ No newline at end of file
+<meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description[page.lang] }} {% endif %}" />
+<meta name="author" content="{% if page.creator %}{{ page.creator }}{% else %}{{ site.creator[page.lang] }} {% endif %}"/>
+<meta name="dcterms.title" content="{{ page.title }}"/>
+<meta name="dcterms.description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description[page.lang] }} {% endif %}"/>
+<meta name="dcterms.creator" content="{% if page.creator %}{{ page.creator }}{% else %}{{ site.creator[page.lang] }} {% endif %}"/>
+<meta name="dcterms.language" title="ISO639-2/T" content="{{ i18nText-language | downcase | truncate: 3, '' }}"/>
```

Comment:
Add a new jekyll property with the 3 letter alternative instead of truncating.

https://github.com/wet-boew/GCWeb/blob/master/sites/includes/i18n-core.liquid#L12